packages/contract-artifacts/test/contract_artifacts_test.ts

```diff
@@ -29,6 +29,10 @@ describe('Contract Artifacts', () => {
         const missingRequiredPropertiesByArtifact: ObjectMap<string[]> = {};
         for (const [artifactName, artifact] of Object.entries(artifacts)) {
             for (const requiredProperty of REQUIRED_PROPERTIES) {
+                // HACK (xianny): Remove after `compiler` field is added in v3.
```

Comment:
```suggestion
                // HACK(xianny): Remove after `compiler` field is added in v3.
```

Reply:
Can you elaborate on when exactly we will remove this check? Also, `schemaVersion` is the version of the artifact schema. So since we're adding `compiler`, shouldn't be bump the schema version to `2.1.0`?

Reply:
discussed 🗣 and agreed that we will keep schemaVersion the same because we haven't changed any compiler output fields. These fields are only changed for publishing on npm so we will bump the npm version when the artifacts are updated next time.